templates/_federalist-redirects.njk

```diff
@@ -196,12 +196,62 @@ server {
 }
 
 
-# guides.18f.gov to github.com/18F/guides-template
+# guides.18f.gov to 18f.gsa.gov/guides/
+# individual guides re-routed to their current
+# addresses, for example:
+#    guides.18f.gov/<guide-name> to <guide-name>.18f.gov
 server {
   listen {{ PORT }};
-  set $target_domain github.com/18F/guides-template;
   server_name guides.18f.gov;
-  return 301 https://$target_domain;
+
+  location ~ ^/?$ {
```

Comment:
[non-blocking question] Is there a reason we want case-sensitive matching? (my understanding is that a tilde = case-sensitive)

Reply:
Hmm, good question. My starting point was another (unrelated) block in that template file that was doing similar things [here](https://github.com/cloud-gov/pages-redirects/pull/221/files#diff-34e945d2103438f007350a2e8042d5c8cba9dbf307b9a92cd964c15d18473da3L517-L537), and they used the tilde. But based on the nginx docs you are absolutely right. 
  
I poked around and found that in [RFC 7230](https://www.rfc-editor.org/rfc/rfc7230), it says that anything in the path should be case sensitive (see [Section 2.7.3](https://www.rfc-editor.org/rfc/rfc7230#section-2.7.3)). So I guess it's standard in HTTP based URIs to use case-sensitive matching for the path parts.

Reply:
Ok, fine by me to keep it that way.